.github/workflows/render-site.yaml

```diff
@@ -0,0 +1,49 @@
+name: Check rendered site for broken links
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Quarto
```

Comment:
I'd try something like this to always grab the latest but you will need to test this actually works: 

```
- name: Set up Quarto
  run: |
    latest_version=$(curl -s https://api.github.com/repos/quarto-dev/quarto-cli/releases/latest | grep 'tag_name' | cut -d\" -f4)
    wget https://github.com/quarto-dev/quarto-cli/releases/download/${latest_version}/quarto-${latest_version}-linux-amd64.deb
    sudo dpkg -i quarto-${latest_version}-linux-amd64.deb
```

Reply:
This one was giving broken links, so I made it check the API site for the whole URL:
```
- name: Get latest Quarto release URL
      id: get-quarto-url
      run: |
        API_URL="https://api.github.com/repos/quarto-dev/quarto-cli/releases/latest"
        DOWNLOAD_URL=$(curl -s $API_URL | jq -r '.assets[] | select(.name | endswith("linux-amd64.deb")).browser_download_url')
        echo "DOWNLOAD_URL=$DOWNLOAD_URL" >> $GITHUB_ENV

    - name: Download and install Quarto
      run: |
        wget ${{ env.DOWNLOAD_URL }} -O quarto-latest-linux-amd64.deb
        sudo dpkg -i quarto-latest-linux-amd64.deb
```